### DISCUSSION TITLE
images and text in page header

### DISCUSSION THREAD

**Discussion by aghaynes** [typst/typst#3495] [Q&A] [answered]
Hi! I'm looking at using typst in conjunction with quarto for creating reports at work. Our corporate identity requires 2 images in the page header, one top left, one top right (similar to the MDPI one pictured on the typst website). I can't figure out how to add two image though... Does anyone know?

**Comment by supcik** [comment]
Do you mean something like that :

![image](https://github.com/typst/typst/assets/30344/dab82f80-3fad-459c-b46b-c8c7398d61c8)

**Reply by supcik** [reply, marked as answer]
I did this with `tablex`, but I guess that a simple `grid` would do it as well. Here is how I did :

``` typst
// Title page : Logo header
block(inset: (top: -30pt),tablex(
  columns: (1fr, 1fr),
  align: (left + horizon, right + horizon),
  auto-lines: false,
  stack(dir: ttb,
    image("img/logo-mse-m0.svg", height: 6mm),
    text(
      stretch: 60%, baseline: 5mm, size: 11pt,
      par(
        leading: 0.5em,
        [Master of Science HES-SO in Engineering\
        Av. de Provence 6\
        CH-1007 Lausanne]
      )
    ),
  ),
image("img/logo-hesso.svg", height: 12mm),
))
```

**Reply by aghaynes** [reply]
exactly! thats great! thanks a lot!

I suppose that all goes in set page (?):

```
#set page(header: locate(loc => {
  if counter(page).at(loc).first() > 1 [
    //second+ head
  ] else [
    //first page head
  ]

})
```

**Reply by supcik** [reply]
I have a function to print the title page. You can read the full document here : https://typst.app/project/rdZYJKiAoBFEjOqIW8nIBr

**Reply by aghaynes** [reply]
Wow! Thank you! Such a great resource!
